client/src/components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components/macro'
+
+const MovieCard = ({ imgUrl, title, year, genre }) => {
+  return (
+    <Wrapper>
+      <img src={imgUrl} alt={title} height="562" width="375" />
+      <h2>{title}</h2>
+      <p>
+        {year} - {genre}
+      </p>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.section`
+  background-color: var(--color-yellow);
+  border: 2px solid var(--color-lila);
+  border-radius: 10px;
+  color: #000;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0.5rem auto;
+  padding: 0.5rem;
+  width: 340px;
+
+  img {
```

Comment:
I got an interesting question here. wouldn't this apply to every img that is in the section, even if it is rendered inside of another sub-component?

Reply:
No, I think it's just in that one section. But I am not 100% sure either.